metadata.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.1",
   "author": "puppetlabs",
   "summary": "Puppet module providing the certregen face for regenerating CA certificates",
-  "license": "Puppet Enterprise Software License Agreement",
+  "license": "proprietary",
```

Comment:
I assume the answer to this question is no given the previous value here. I have updated the PR with the followiing changes:
1. The NOTICE file has been deleted
2. The LICENSE file has been replaced with the `Puppet Enterprise Software License Agreement`
These changes bring the licensing in parity with the [sqlserver module](https://github.com/puppetlabs/puppetlabs-sqlserver)